### PR TITLE
adding graphql page for rich text example

### PR DESCRIPTION
**Pull request by NoahBluescape**

### CODE REVIEW

File: site-templates/pages/graphql-rich-text.gomd [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+            blocks {
+                ...on TextBlock {
+                align content}}
+                }
+            }
+    }
+</pre>
+
+<p>
+This mutation can be called with the following example payload:
```
Comment (vlad-bluescape) [inline site-templates/pages/graphql-rich-text.gomd]: For consistency with other pages, please change to "This mutation can be called with the following variables"

File: site-templates/pages/graphql-rich-text.gomd [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+            }
+    }
+</pre>
+
+<p>
+This mutation can be called with the following example payload: 
+</p>
+
+<pre>
+"workspaceId": workspaceId,
```
Comment (vlad-bluescape) [inline site-templates/pages/graphql-rich-text.gomd]: Add the angled brackets, as on /docs/page/graphql-subscriptions: <workspaceId>
image

File: site-templates/pages/graphql-rich-text.gomd [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+</p>
+
+<p>
+<pre class="lang lang-python">
+import requests
+import pprint
+
+url = "https://elementary.stg1.bluescape.com/graphql/"
+workspaceId = "N-DUJ6t1OAopxsQQqTi7"
+token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6ImswIn0.eyJleHAiOjE2MzEyMTIxMjcsInNwaWQiOm51bGwsIm5iZiI6MTYzMTEyNTcxNywiaWF0IjoxNjMxMTI1NzI3LCJqdGkiOiJBQTVYd0dHODJZR0hydloyOW1abng1TEZyRDdvZ0JJallDemNKYm54em93NklWaFUiLCJzdWIiOiJ0ZTRESVRUNmo0NlJ6bkVWVDF5bCIsInVzZXJUeXBlIjoicmVnaXN0ZXJlZCIsImF1ZCI6WyJpc2FtZG5qSnNrakdHNER6ZWQyWnZJZFRYU0E4bkRVNjRQS3NhIiwiMzZmOGNmNTE3NWU0ZmFhYTRmMDY3MTg0MDRiN2RmOTRkZGMwZDhhZSIsIjAxNWY0ODFhY2ZlNDAyYzliYWU0MzUzODllZmJiNjkxNDkyOWNlOWQiLCIwMTVmNDgxYWNmZTQwMmM5YmFlNDM1Mzg5ZWZiYjY5MTQ5MjliZjhjIiwiVkpCdGlhSmRuakota2pHRzREemVkMlp2SWRUWFNBOG5EVTY0UEswYyJdLCJpc3MiOiJodHRwczovL2lkZW50aXR5LWFwaS5zdGcxLmJsdWVzY2FwZS5jb20ifQ.CnOIGsEWuFjRKyIv-LVMKf7dNco9f8qHzsKpK0TOHPk"
```
Comment (vlad-bluescape) [inline site-templates/pages/graphql-rich-text.gomd]: Remove token, as in the other examples: use &lt;TOKEN&gt;

File: site-templates/pages/graphql-rich-text.gomd [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+</pre>
+</p>
+
+<p>
+<pre class="lang lang-python">
+import requests
+import pprint
+
+url = "https://elementary.stg1.bluescape.com/graphql/"
+workspaceId = "N-DUJ6t1OAopxsQQqTi7"
```
Comment (vlad-bluescape) [inline site-templates/pages/graphql-rich-text.gomd]: Remove workdspace ID, use &lt;workspaceID&gt;

File: site-templates/pages/graphql-rich-text.gomd [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+                                },
+                                "backgroundColor": {
+                                    "r": 223,
+                                    "g": 156,
+                                    "b": 200,
+                                    "a": 1
+                                },
+                                "content": [
+                                    {
+                                        "text": "NoahText2"
```
Comment (vlad-bluescape) [inline site-templates/pages/graphql-rich-text.gomd]: Maybe use "Rich Text test" instead

File: site-templates/pages/graphql-rich-text.gomd [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+    pprint.pprint(response)
+</pre>
+</p>
+
+<h3>Output</h3>
+<pre>
+{
+    "data": {
+        "createText": {
+            "id": "613ba16421acb112a55a5753",
```
Comment (vlad-bluescape) [inline site-templates/pages/graphql-rich-text.gomd]: woudl change with &lt;object_ID&gt; , JUST In case somebody expects to get this ID

**Review by vlad-bluescape — CHANGES_REQUESTED** [review]

File: site-templates/pages/graphql-rich-text.gomd (hunk 3 above)
Comment (NoahBluescape) [reply site-templates/pages/graphql-rich-text.gomd]: Oops 😃

File: site-templates/pages/graphql-rich-text.gomd (hunk 1 above)
Comment (NoahBluescape) [reply site-templates/pages/graphql-rich-text.gomd]: I am changing this to be consistent for now, but I think I'd like to change the language on all the pages to be more descriptive to make it clear this is just an "example" of the variables that you can use, but they are not all required.

File: site-templates/pages/graphql-rich-text.gomd (hunk 4 above)
Comment (NoahBluescape) [reply site-templates/pages/graphql-rich-text.gomd]: Oops again! and the URL! Looks like I missed pruning this one.

File: site-templates/pages/graphql-rich-text.gomd (hunk 5 above)
Comment (NoahBluescape) [reply site-templates/pages/graphql-rich-text.gomd]: Dang, all the others had "Rich Text Example". MIssed this one.

File: site-templates/pages/graphql-rich-text.gomd (hunk 1 above)
Comment (vlad-bluescape) [reply site-templates/pages/graphql-rich-text.gomd]: Here it makes sense to say it is an example, on other mutations/queries it is more direct: "this is what you need to use".

**Pull request merged**